go.mod

```diff
@@ -31,19 +31,18 @@ require (
 	go.opentelemetry.io/otel/metric v1.26.0
 	go.opentelemetry.io/otel/sdk/metric v1.26.0
 	go.uber.org/zap v1.27.0
-	golang.org/x/crypto v0.22.0
-	golang.org/x/oauth2 v0.19.0
-	google.golang.org/api v0.170.0
-	google.golang.org/grpc v1.62.1
-	google.golang.org/protobuf v1.33.0
+	golang.org/x/crypto v0.23.0
```

Comment:
It seems these go.mod upgrades are not really required for this PR to work, therefore we should remove those changes (go.mod and go.sum)